pkg/merkhet/merkhet_suite_test.go

```diff
@@ -103,12 +107,12 @@ func (l *LoggerMock) ID() int {
 	return 0
 }
 
-func (l *LoggerMock) Write(p []byte) (n int, err error) {
+func (l *LoggerMock) Write(p []byte, level logger.LogLevel) (n int, err error) {
```

Comment:
May I suggest to swap the arguments, it feels weird to have the log content first, and the LogLevel second.